activities/models/post.py

```diff
@@ -141,6 +164,7 @@ class urls(urlman.Urls):
         action_unboost = "{view}unboost/"
         action_delete = "{view}delete/"
         action_reply = "/compose/?reply_to={self.id}"
+        action_edit = "/compose/{self.id}/edit/"
```

Comment:
This URL should probably either be `{view}edit/` or `/compose/?edit={self.id}`. Any preference?

Reply:
Switching to `{view}edit/`